[[source]]
name = "pypi"
url = "https://pypi.org/simple"
verify_ssl = true

[dev-packages]
atomicwrites = "==1.4.0"
boto3 = "==1.17.16"
cython = "==0.29.22"
freezegun = "==1.0.0"
pre-commit = "==2.10.1"
pylint = "==2.7.1"
pytest = "==6.2.2"
pytest-cov = "==2.11.1"
pytest-mock = "==3.5.1"
pytest-profiling = "==1.7.0"
pytest-pylint = "==0.18.0"
pytest-randomly = "==3.5.0"
pytest-timeout = "==1.4.2"
sphinx = "==3.5.1"

[packages]
astroid = "==2.4.2"  # astroid 2.5.0 was causing issues with pylint, so pinning it to 2.4.2 until the issue is resolved in a new version
flask = "==1.1.2"
flask-cors = "==3.0.10"
flatten-dict = "==0.3.0"
h5py = "==3.1.0"
immutabledict = "==1.2.0"
immutable-data-validation = "==0.2.1"
labware-domain-models = "==0.3.1"
nptyping = "==1.4.0"
numpy = "==1.20.1"
mantarray-file-manager = "==0.4.5"
mantarray-waveform-analysis = "==0.5.11"
scipy = "==1.6.1"
secrets-manager = "==0.4"
semver = "==2.13.0"
stdlib-utils = "==0.4.2"
xem-wrapper = "==0.2.5"

[requires]
python_version = "3.8"